### PR TITLE
Testing config

### PR DESCRIPTION
**Pull request by adumortier**
## Who worked on this

- [x] Alexis
- [ ] Maria
- [ ] Ed
- [ ] Cody

## Type of change

- [ ] Style (Non-breaking Style code)
- [x] Testing (Non-breaking Testing code)
- [ ] Bug fix or refactor (change which fixes an issue)
- [ ] New feature (change which adds functionality)

## Summary of the change.

- changed the file organization to separate the testing and development config
- changed the path to the modules 

## Checklist:

- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code if necessary

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: gunicorn --pythonpath trackbasket_be app:app
+web: gunicorn run:app
```
Comment (adumortier) [inline Procfile:1]: the `app` is included in the `run.py` file

File: tests/test_at_risk_user.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+from trackbasket_be.models.at_risk_user import AtRiskUser
+from trackbasket_be import create_app, db
+
+class TestAtRiskUser(unittest.TestCase):
+
+  def setUp(self):
+    self.app = create_app('testing')
+    self.test_app = self.app.test_client()
+    with self.app.app_context():
+        db.create_all()
```
Comment (adumortier) [inline tests/test_at_risk_user.py:13]: creates an app and a database for testing purposes

File: tests/test_shopping_list.py [hunk 3]
```diff
@@ -6,22 +6,22 @@
 from trackbasket_be.models.item import Item
 from trackbasket_be.models.shopping_list import ShoppingList
 from trackbasket_be.models.store import Store
-from trackbasket_be.app import app
+from trackbasket_be import create_app, db
```
Comment (adumortier) [inline tests/test_shopping_list.py:9]: the `create_app` method and `db` are imported from `__init__.py`

File: tests/test_store.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    self.app = create_app('testing')
+    self.test_app = self.app.test_client()
+    with self.app.app_context():
+      db.create_all()
+    self.store = Store(name='store1', location_id='9820340237', address= '12 Elm Street', city= 'Cambridge', state= 'MA', zipcode= '02139', latitude= '1234', longitude= '4321')
+  
+  def tearDown(self):
+    with self.app.app_context():
+      db.session.remove()
+      db.drop_all()
```
Comment (adumortier) [inline tests/test_store.py:18]: adds a `teardown` method to clear the testing database at the end of each test

File: tests/test_store.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  #   payload = json.loads(response.data)
+    
+  #   self.assertEqual(payload['location_id'], '62000010')
+  #   self.assertEqual(payload['name'], 'KINGSOOPERS')
+  #   self.assertEqual(payload['address'], '15250 E Mississippi Ave')
+  #   self.assertEqual(payload['city'], 'Aurora')
+  #   self.assertEqual(payload['state'], 'CO')
+  #   self.assertEqual(payload['zipcode'], '80012')
+  #   self.assertEqual(payload['latitude'], 39.6951161)
+  #   self.assertEqual(payload['longitude'], -104.8109163)
```
Comment (adumortier) [inline tests/test_store.py:42]: this will need to be uncommented in next PR

File: trackbasket_be/resources/at_risk_user.py [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
-from models.at_risk_user import AtRiskUser
-from models.basemodel import BaseModel,db 
-from models.krogerservice import Krogerservice
-from models.store import Store
-from models.item import Item
+from ..models.at_risk_user import AtRiskUser
+from ..models.basemodel import BaseModel,db 
+from ..models.krogerservice import Krogerservice
+from ..models.store import Store
+from ..models.item import Item
```
Comment (adumortier) [inline trackbasket_be/resources/at_risk_user.py:6]: changes the path to the modules of interest

File: trackbasket_be/__init__.py [hunk 7]
```diff
@@ -1 +1,52 @@
...
+def create_app(config_name):
+
+  from flask import Flask, request
+  from flask_restful import Resource, Api
+  from trackbasket_be.config import app_config
+
+
+  app = Flask(__name__)
+  api = Api(app)
+  app.config.from_object(app_config[config_name])
```
Comment (mronauli) [inline trackbasket_be/__init__.py:16]: loads environment based on app config key

File: trackbasket_be/config.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+class TestingConfig(Config):
+    TESTING = True
+    DEBUG = True
+    BUNDLE_ERRORS = True
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///testing.db'
+
+
+class ProductionConfig(Config):
+    DEBUG = False
+    TESTING = False
```
Comment (mronauli) [inline trackbasket_be/config.py:23]: creating class for each environment that inherits from Config class

**Pull request merged**